Fix #1840: Introduce a nicer model for configuring network presentation. (#1848)
kroxylicious/kroxylicious: docs/modules/configuring/ref-configuring-proxy-example.adoc

<6> The name of the virtual cluster.
<7> The bootstrap address of the target physical Kafka Cluster being proxied.
<8> TLS configuration for the connection to the target cluster.
<9> The listeners of the virtual cluster defining how the virtual cluster is presented to the network.

> <10> The identification scheme used to route traffic to brokers - the possibilities are `sniHostIdentifiesNode` and `portIdentifiesNode`.

Reviewer: ```suggestion
<10> The identification scheme used to route traffic to brokers, which can be `sniHostIdentifiesNode` or `portIdentifiesNode`.
```

<11> The hostname and port of the bootstrap used by the Kafka clients to connect to this listener. The hostname must be resolved by the clients.

> <12> TLS encryption by this listener for securing connections with the clients.

Reviewer: ```suggestion
<12> TLS encryption used by the listener for securing connections with the clients.
```
